cap/modules/workflows/serializers.py

```diff
@@ -42,6 +45,29 @@ class ReanaWorkflowSchema(Schema):
     status = fields.Str(dump_only=True)
     service = fields.Str(dump_only=True)
     workflow_json = fields.Dict(dump_only=True)
+    links = fields.Method('build_links', dump_only=True)
+
+    def build_links(self, obj):
+        """Construct schema links."""
+        def url_with_wf(path):
+            return url_to_api_url(
+                url_for(path, workflow_id=obj['workflow_id']))
+
+        links = {
+            'self': url_with_wf('cap_workflows.get_workflow'),
+            'all': url_to_api_url(url_for(
+                'cap_workflows.get_all_reana_workflows', depid=obj['depid'])),
+            'create':
+                url_to_api_url(url_for('cap_workflows.create_reana_workflow')),
+            'clone': url_with_wf('cap_workflows.clone_reana_workflow'),
+            'start': url_with_wf('cap_workflows.start_reana_workflow'),
+            'stop': url_with_wf('cap_workflows.stop_reana_workflow'),
+            'delete': url_with_wf('cap_workflows.delete_reana_workflow'),
+            'files': url_with_wf('cap_workflows.list_reana_workflow_files'),
+            'status': url_with_wf('cap_workflows.get_reana_workflow_status'),
+            'logs': url_with_wf('cap_workflows.get_reana_workflow_logs')
+        }
+        return links
 
 
 class ReanaAllWorkflowsSchema(Schema):
```

Comment:
I'm not sure why we would need this one (and indeed it's not used anywhere), can we remove it?